test/plugins/locale.test.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable dot-notation */
+import { esday } from 'esday'
+import { describe, expect, it } from 'vitest'
+import localeZh from '~/locales/zh'
+import { localePlugin } from '~/plugins/locale'
+
+esday.extend(localePlugin)
+
+describe('factory locale methods', () => {
+  it('set locale', () => {
+    esday.locale('zh')
+    // @ts-expect-error $l is private property
+    expect(esday()['$l']).toBe('zh')
```

Comment:
This test shows that the getter for the locale is missing -the internal variable `$l` is an **internal** variable.

Reply:
I'll create a new method to get locale name

Reply:
It is a internal variable. I don't understant what's the issue here.

Reply:
IMHO this test shows that a getter for the currently active locale has its use cases and that perhaps it would make sense to add one.

I don't think that in a real life application using this package internal variables should be used.